pkg/pb/metainfo.proto

```diff
@@ -24,6 +24,7 @@ service Metainfo {
     // Object
     rpc BeginObject(ObjectBeginRequest) returns (ObjectBeginResponse);
     rpc CommitObject(ObjectCommitRequest) returns (ObjectCommitResponse);
+    rpc GetObject(ObjectGetRequest) returns (ObjectGetResponse);
```

Comment:
I don't recall this from the original discussion re RPC methods. What is the reasoning here?

Reply:
yeah, we missed that one. We need a method to access an object in general and to have object `stream_id` for segment methods like e.g. `ListSegments`. See https://storj.slack.com/archives/CCXRMC54M/p1563789833001000